### PR TITLE
[fix] lightnovelbrasil

### LINKED ISSUE #910: Fix Light Novel Brasil
### Source information

Light Novel Brasil 

### Steps to reproduce

novels don't appear when open the extension
Screenshot_20240426_151338_LNReader

### Expected behavior

it should return the novels 

### Actual behavior

Screenshot_20240426_151338_LNReader

nothing happens instead

### LNReader version

2.0.0 beta 1

### Android version

Android 13

### Other details

when i search for a novel, it appears and work fine...
Screenshot_20240426_151627_LNReader

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.19](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by error7404**
closes #910

### CODE REVIEW

File: scripts/multisrc/lightnovelwp/sources.json [hunk 1]
```diff
@@ -166,5 +166,14 @@
...
     }
+  },
+  {
+    "id": "lightnovelbrasil",
+    "sourceSite": "https://lightnovelbrasil.com/",
+    "sourceName": "Light Novel Brasil",
+    "options": {
+      "lang": "Portuguese",
+      "reverseChapters": true
+    }
```
Comment (nyagami) [inline scripts/multisrc/lightnovelwp/sources.json:177]: this need versionIncrements

Comment (error7404) [reply scripts/multisrc/lightnovelwp/sources.json:177]: It was a separate file before so it was version 1.0.0 before

Comment (nyagami) [reply scripts/multisrc/lightnovelwp/sources.json:177]: the old one didnt work, does this work?

Comment (error7404) [reply scripts/multisrc/lightnovelwp/sources.json:177]: Yes

Comment (nyagami) [reply scripts/multisrc/lightnovelwp/sources.json:177]: oh, I forgot this template version is 1.0.4

**Review by nyagami — APPROVED** [review]

**Pull request merged**